azure/servicemanagement/__init__.py

```diff
@@ -1873,6 +1876,59 @@ def parse_response_for_async_op(response):
     return result
 
 
+def get_certificate_from_publish_settings(publish_settings_path, path_to_write_certificate, subscription_name=None):
+    '''
+    Writes a certificate file to the specified location.  This can then be used 
+    to instantiate ServiceManagementService.  Returns the subscription ID.
+
+    publish_settings_path: 
+        Path to subscription file downloaded from 
+        http://go.microsoft.com/fwlink/?LinkID=301775
+    path_to_write_certificate:
+        Path to write the certificate file.
+    subscription_name:
+        (optional)  Provide a subscription name here if you wish to use a 
+        specific subscription under the publish settings file.
+    '''
+    import base64
+    import OpenSSL.crypto as crypto
```

Comment:
This is a relatively expensive import so I put it in the def to avoid unnecessary loading